Dockerfile

```diff
@@ -2,19 +2,21 @@ FROM eclipse-temurin:17
 
 
 # Update
-RUN apt-get update
 RUN apt-get install -y wget
 
 # Set up file system
-
 RUN mkdir src
 RUN mkdir dependency
 
-# Download VerveineJ
+# Creating executable
+RUN /bin/sh -c "echo \"#!/bin/sh\" > /verveinej-docker.sh"
+RUN /bin/sh -c "echo \"/VerveineJ-3.0.7/verveinej.sh -autocp /dependency $* .\" >> /verveinej-docker.sh"
```

Comment:
I think you should move the `$*` before the `-autocp /dependency`to allow people using Java option such as ` -Xmx8g -Xms8g --`